src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -41,6 +42,7 @@ public class DocumentBuilder : IDisposable
         public DocumentBuilder(
             IEnumerable<Assembly> assemblies,
             ImmutableArray<string> postProcessorNames,
+            string templateHash,
```

Comment:
>templateHash [](start = 19, length = 12)

also give `templateHash` a default value `null`, and in this way it doesn't need change unit test.